model/src/main/proto/screens.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+enum ScreenName {
+  ACTIVITY_NAME_UNSPECIFIED = 0;
```

Comment:
Should be SCREEN_NAME_UNSPECIFIED (per convention).

Reply:
done.